cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1707,6 +1729,11 @@ private void intraBrokerMoveReplicas() {
         waitForIntraBrokerReplicaTasksToFinish();
         inExecutionTasks = inExecutionTasks();
       }
+
+      if (_logDirThrottle != null) {
```

Comment:
for my understanding, why the underscore in the variable names here?

Reply:
In consistency with `_replicationThrottle`.